modules/ppcp-button/src/Assets/SmartButton.php

```diff
@@ -1350,6 +1352,51 @@ protected function product_supports_payment( WC_Product $product ): bool {
 		);
 	}
 
+	/**
+	 * Checks if PayPal buttons/messages should be rendered for the current page.
+	 *
+	 * @param string|null $context The context that should be checked, use default otherwise.
+	 *
+	 * @return bool
+	 */
+	protected function is_button_disabled( string $context = null ): bool {
+		if ( null === $context ) {
```

Comment:
We removed yoda condition code style so no need to use it anymore, same for `if ( 'product' === $context ) {` below.

Reply:
I don't mind, and actually I usually use it when comparing with null. Left the same.